client/websocket/websocket.go

```diff
@@ -257,9 +256,11 @@ out:
 				break out
 			}
 		case <-ctx.Done():
-			break out
+			// We are skipping m.feed.Close if the feed were closed (external sig).
+			return
 		}
 	}
+	m.feed.Close()
```

Comment:
Following the convo at https://github.com/decred/dcrdex/commit/361efe5a95256c57aa1b15eb1374ba68aba3006c#r48311533, I called out the wrong case for return with no `m.feed.Close()`.  We skip it in `case update, ok := <-m.feed.C` where `!ok`.  But also be sure that only the one case (external signal to close) does return and the others do `break out`.  Presently there's another return